src/ResourceManager/IotCentral/Commands.IotCentral/Commands.IotCentral.csproj

```diff
@@ -0,0 +1,116 @@
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
+    <OutputPath>..\..\..\Package\Release\ResourceManager\AzureResourceManager\AzureRM.IotCentral\</OutputPath>
+    <DefineConstants>TRACE;SIGN</DefineConstants>
+    <Optimize>true</Optimize>
+    <DebugType>pdbonly</DebugType>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>true</DelaySign>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Common\FullParameterSetCmdlet.cs" />
+    <Compile Include="Common\IotCentralBaseCmdlet.cs" />
+    <Compile Include="Common\IotCentralUtils.cs" />
+    <Compile Include="IotCentral\GetAzureRmIotCentralApp.cs" />
+    <Compile Include="IotCentral\NewAzureRmIotCentralApp.cs" />
+    <Compile Include="IotCentral\RemoveAzureRmIotCentralApp.cs" />
+    <Compile Include="IotCentral\SetAzureRmIotCentralApp.cs" />
+    <Compile Include="Models\PSIotCentralApp.cs" />
+    <Compile Include="Models\PSIotCentralAppSku.cs" />
+    <Compile Include="Models\PSIotCentralAppSkuInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Properties\Resources.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+      <CustomToolNamespace>Microsoft.Azure.Commands.Management.IotCentral.Properties</CustomToolNamespace>
+      <SubType>Designer</SubType>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="AzureRM.IotCentral.psd1">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="help\AzureRM.IotCentral.md" />
+    <None Include="help\Get-AzureRmIotCentralApp.md" />
+    <None Include="help\New-AzureRmIotCentralApp.md" />
+    <None Include="help\Remove-AzureRmIotCentralApp.md" />
+    <None Include="help\Set-AzureRmIotCentralApp.md" />
+    <None Include="MSSharedLibKey.snk" />
+    <None Include="StartupScripts\*.ps1" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.IotCentral, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.IotCentral.1.0.0\lib\net452\Microsoft.Azure.Management.IotCentral.dll</HintPath>
+    </PackageReference>
+    <PackageReference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Remove these two clientruntime reference - we do this in common so we can update cleanly across the repo.

Reply:
Done